assets/src/components/detours/diversionPage.tsx

```diff
@@ -59,6 +60,7 @@ export const DiversionPage = ({
           onClickOriginalShape={addConnectionPoint}
           undoDisabled={canUndo === false}
           onUndo={undo}
+          onClear={clear}
```

Comment:
**Thought:** This is exactly the kind of annoying little diff that we can get away from if we pursue @firestack's idea and make this whole thing more composable.